examples/compute/instance/basic/main.tf

```diff
@@ -23,6 +23,12 @@ resource "vkcs_compute_instance" "basic" {
   security_groups = [
     vkcs_networking_secgroup.admin.name
   ]
+  # or you may set security group by id. 
```

Comment:
we're adding this argument to propose users to use it instead of the old one

Reply:
done